service/gateway/gateway_config.go

```diff
@@ -17,4 +17,5 @@ type GatewayConfig struct {
 	ReceiverServiceAddress   string
 	MetadataServiceAddress   string
 	AuthServiceAddress       string
+	ApiLimiterConfig         *ApiLimiterConfig
```

Comment:
fix `ApiLimiterConfig         *ApiLimiterConfig` to `APILimiterCfg         *APILimiterConfig`